src/commands/__tests__/cursorNext.ts

```diff
@@ -75,25 +75,17 @@ describe('normal view', () => {
     store.dispatch([
       importText({
         text: `
-          - SORT
             - a
               - a1
             - c
             - b`,
       }),
-      (dispatch, getState) =>
-        dispatch(
-          toggleAttribute({
-            path: contextToPath(getState(), ['SORT']),
-            values: ['=sort', 'Alphabetical'],
-          }),
-        ),
-      setCursor(['SORT', 'a']),
-      cursorNext(),
     ])
-
+    setCursor(['a'])
+    executeCommandWithMulticursor(toggleSortShortcut, { store })
```

Comment:
Great! This is better because it triggers the toggleSort shortcut rather than directly setting meta attributes.